tests/runtests/mailx-dead-letter/runtest.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# vim: dict=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   runtest.sh of mailx-reporting
+#   Description: Tests abrt-action-mailx
+#   Author: Michal Nowak <mnowak@redhat.com>
+# # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright (c) 2011 Red Hat, Inc. All rights reserved.
+#
+#   This copyrighted material is made available to anyone wishing
+#   to use, modify, copy, or redistribute it subject to the terms
+#   and conditions of the GNU General Public License version 2.
+#
+#   This program is distributed in the hope that it will be
+#   useful, but WITHOUT ANY WARRANTY; without even the implied
+#   warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#   PURPOSE. See the GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public
+#   License along with this program; if not, write to the Free
+#   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+#   Boston, MA 02110-1301, USA.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+. /usr/share/beakerlib/beakerlib.sh
+. ../aux/lib.sh
+
+TEST="mailx-dead-letter"
+PACKAGE="abrt"
+
+rlJournalStart
+    rlPhaseStartSetup
+        check_prior_crashes
+
+        TmpDir=$(mktemp -d)
+        cp mailx.conf $TmpDir
+        rlRun "pushd $TmpDir"
+    rlPhaseEnd
+
+    rlPhaseStartTest
+        # missing sendmail causes that mailx fails
+        SENDMAIL=$(which sendmail)
+        rlFileBackup $SENDMAIL
+        rlRun "mv -f $SENDMAIL ${SENDMAIL}.backup"
+
+        prepare
+        generate_crash
+        get_crash_path
+        wait_for_hooks
+
+        rlRun "ls $crash_PATH > dir_content.log"
+
+        rlRun "reporter-mailx -d $crash_PATH -c mailx.conf" 1
```

Comment:
It creates ``reported_to`` file only in the case ``reporter-mailx`` sent the mail successfully. ``reporter-mailx``, In this case, should fail, so checking not creating ``reported-to`` file is desirable.